share/commands/agent-server-keys-reset

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+# @description Reset all keys known from this node.
+# @man This command will delete all known keys from this node, allowing it to connect to
+# @man another server or relay by trusting it.
+# @man *Options*:
+# @man +
+# @man *-f*: force the reset of all keys without asking for confirmation
+# @man +
+# @man *-v*: run the command in verbose mode
+
+. "${BASEDIR}/../lib/common.sh"
+
+VERBOSE=false
+FORCE=0
+OPTS=""
+
+
+while getopts "vf" opt; do
+  case $opt in
+    v)
+      VERBOSE=true
+      ;;
+    f)
+      FORCE=1
+      ;;
+  esac
+done
+
+if [ "${UUID}" = "root" ]
+then
+  echo "Reinitializing all known keys on the root server breaks all agents. ABORTING!" >&2
```

Comment:
I don't think it would break agents connection, server always trusts everybody. But it actually makes no sense as server only trust itself for remote-run or policy/file copy.